Presentation.Web/Controllers/API/V2/Internal/Users/Mapping/UserResponseModelMapper.cs

```diff
@@ -24,5 +25,24 @@ public UserResponseDTO ToUserResponseDTO(User user)
                 LastSentAdvis = user.LastAdvisDate
             };
         }
+        public UserWithIsPartOfCurrentOrgResponseDTO ToUserWithIsPartOfCurrentOrgResponseDTO(Guid organizationUuid, User user)
+        {
+            return new UserWithIsPartOfCurrentOrgResponseDTO
+            {
+                Uuid = user.Uuid,
+                Email = user.Email,
+                FirstName = user.Name,
+                LastName = user.LastName,
+                PhoneNumber = user.PhoneNumber,
+                DefaultUserStartPreference =
+                    DefaultUserStartPreferenceChoiceMapper.GetDefaultUserStartPreferenceChoice(
+                        user.DefaultUserStartPreference),
+                HasApiAccess = user.HasApiAccess,
+                HasStakeHolderAccess = user.HasStakeHolderAccess,
+                Roles = user.OrganizationRights.Select(x => x.Role.ToOrganizationRoleChoice()).ToList(),
```

Comment:
I'm not sure what is intented here, but if (A) this should only contain roles of the queried organization, then use the user.GetRolesInOrganization(orgUuid) here, if instead (B) this should contain roles in all organizations, then i think we need to return OrganizationRights instead of Roles here, as a OrganizationRole doesn't contain information about which organization it's tied to.